src/com/mitrais/atm/Main.java

```diff
@@ -1,9 +1,20 @@
 package com.mitrais.atm;
 
+import com.mitrais.atm.repository.AccountRepository;
 import com.mitrais.atm.screen.WelcomeScreen;
 
 public class Main {
     public static void main(String[] args) {
+        loadAccountData(args);
         WelcomeScreen.getInit().show();
     }
+
+    private static void loadAccountData(String[] args) {
+        if(args.length < 1) {
+            System.out.println("📣 Please include with filePath of the CSV data source file.");
```

Comment:
If your program can't find a file, then load the default accounts from previous stage

Reply:
I have fixed it, now It will be loaded default data by changing at this 8069117